index.js

```diff
@@ -33,6 +33,16 @@ const OptionsType = new Archetype({
     $type: 'string',
     $required: false,
     $default: null
+  },
+  expiresColumn: {
```

Comment:
I'd call this `expiresKey` to be a bit more consistent with mongodb's nomenclature

Reply:
Ok